content/updates/2020-11/1-Stop-your-imports-and-exports.md

```diff
@@ -0,0 +1,24 @@
+---
+pim_announcement_alt_img: "The new Stop button in the process tracker"
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Stop your imports and exports
+::: meta-data type="New" features="Productivity" available="mid-November" link-to-doc="../articles/TODO.html"
+:::
+
+A brand new `Stop` button is now displayed in your process tracker. It enables you to stop the Excel and CSV imports/exports you launched. Super helpful whenever you launch a job by mistake and don't want to wait for it to execute until the end. 😉
+
+![The new `Stop` button in the process tracker](../img/TODO.png)
+
+This button comes with a dedicated permission `Stop an ongoing job`. If you need it, ask your PIM admin to give you the right to do so.
+
+On a side note, some other jobs such as mass actions or rule executions can also be stopped at any time in the process tracker. So neat!
+
+::: more
```

Comment:
Maybe we can add links that are not related to imports/exports? Maybe a link related to the setup of permissions, for imports/exports this will be managed in the export profiles, but @felix-couprie how admins can allow PIM users to stop mass actions and rules execution? TY

Reply:
I'll change the links to refer to jobs instead of just imports/exports.

Regarding the permission, it's in the user roles.
I'll add the link to how to set up user roles.